src/EnergyPlus/HVACManager.cc

```diff
@@ -2022,7 +2022,7 @@ void ResolveAirLoopFlowLimits(EnergyPlusData &state)
 
     for (AirLoopIndex = 1; AirLoopIndex <= state.dataHVACGlobal->NumPrimaryAirSys; ++AirLoopIndex) {          // loop over the primary air loops
         for (SupplyIndex = 1; SupplyIndex <= AirToZoneNodeInfo(AirLoopIndex).NumSupplyNodes; ++SupplyIndex) { // loop over the air loop supply outlets
```

Comment:
`auto &thisAirToZoneNodeInfo = state.dataAirLoop->AirToZoneNodeInfo(AirLoopIndex);` here would be more useful than `auto &AirToZoneNodeInfo` above.